public/data/dmitrii-kilishek.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "Dmitrii Kilishek",
+  "type": "personal",
+  "bio": "Hey dear visitor! I'm Dmitrii. A 🇺🇸 based Software Tester, Lifelong Learner, Former Sysadmin, In IT Since 1994, From: 🇷🇺 with ❤️",
```

Comment:
```suggestion
  "bio": "Hey dear visitor! I'm Dmitrii. A US based Software Tester, Lifelong Learner, Former Sysadmin, In IT Since 1994, From: 🇷🇺 with ❤️",
```

Reply:
It can also be made United States or U.S if you want, but for now, it can remain as it is, but just capitalize it.

Reply:
If you think that it makes more sense like this... why not.